features-json/css-paint-api.json

```diff
@@ -197,7 +197,7 @@
       "11":"n",
       "11.1":"n",
       "12":"n",
-      "TP":"n"
+      "TP":"a d #1"
```

Comment:
Change **a** to **n** since **a d** means without any flag enabled, it has partial support